test/python/compiler/test_assembler.py

```diff
@@ -353,6 +364,75 @@ def test_circuit_with_global_phase(self):
         self.assertEqual(getattr(qobj.experiments[0].header, 'global_phase'),
                          .3 * np.pi)
 
+    def test_pulse_gates_single_circ(self):
```

Comment:
Is there a test for calibrations on one circuit and no calibrations at all on another circuit?